playbooks/os/openshift-cluster/launch.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Launch instance(s)
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars_files:
+  - vars.yml
+  vars:
+    image_url: "{{ deployment_vars[deployment_type].image.url }}"
+    image_sha256: "{{ deployment_vars[deployment_type].image.sha256 }}"
+    image_name: "{{ deployment_vars[deployment_type].image.name }}"
+  tasks:
+  - fail:
+      msg: "Deployment type not supported for OpenStack provider yet"
+    when: deployment_type == 'online'
```

Comment:
This should probably be changed to:
```
when: deployment_type in ['online', 'enterprise']
```
Since enterprise deployments will not currently work (because of lack of registration support).